Adyen/UI/Form/FormViewController.swift

```diff
@@ -198,17 +154,6 @@ open class FormViewController: UIViewController, Localizable {
         return super.resignFirstResponder()
     }
     
-    @objc private func keyboardWillChangeFrame(_ notification: NSNotification) {
-        guard let keyboardFrame = notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect else { return }
-        var heightOffset = keyboardFrame.origin.y - UIScreen.main.bounds.height
-        
-        if #available(iOS 11.0, *) {
-            heightOffset += min(abs(heightOffset), view.safeAreaInsets.bottom)
-        }
-        
-        bottomConstraint?.constant = heightOffset
-    }
-    
```

Comment:
why is this removed?

Reply:
I am still hovering on this one as well..

IN one hand: 
We have too many sources of trust for keyboard events. 
Too many things happening in the same time in different parts of the app and it is not easy to control animation.

In the other:
This will lead to changes in Component integration. Also previously we had a white "background" under keyboard, I will have to do something extra.